e2e/tests/functional/plugins/lad/lad.e2e.spec.js

```diff
@@ -0,0 +1,104 @@
+
+test.describe('Testing LAD table @unstable', () => {
+    let sineWaveObject;
+    test.beforeEach(async ({ page, openmctConfig }) => {
+        await page.goto('./', { waitUntil: 'networkidle' });
+        await setRealTimeMode(page);
+        sineWaveObject = await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Test Sine Wave Generator"
+        });
+
+        const { myItemsFolderName } = openmctConfig;
+        await page.locator(`text=${myItemsFolderName}`).first().click();
+    });
+    test('telemetry value exactly matches latest telemetry value received in real time', async ({ page }) => {
+        await createDomainObjectWithDefaults(page, {
+            type: 'LAD Table',
+            name: "Test LAD Table"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
+        await page.dragAndDrop('text=Test Sine Wave Generator', '.c-lad-table-wrapper');
+
+        await page.locator('button[title="Save"]').click();
+        await page.locator('text=Save and Finish Editing').click();
+
+        const getTelemValuePromise = await subscribeToTelemetry(page, sineWaveObject.uuid);
+        const subscribeTelemValue = await getTelemValuePromise;
+        const ladTableValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const ladTableValue = await ladTableValuePromise.textContent();
+
+        expect(ladTableValue).toBe(subscribeTelemValue);
+    });
+    test('telemetry value exactly matches latest telemetry value received in fixed time', async ({ page }) => {
+        await createDomainObjectWithDefaults(page, {
+            type: 'LAD Table',
+            name: "Test LAD Table"
+        });
+        await page.locator('[title="Edit"]').click();
+
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').click();
+        await page.dragAndDrop('text=Test Sine Wave Generator', '.c-lad-table-wrapper');
+
+        await page.locator('button[title="Save"]').click();
+        await page.locator('text=Save and Finish Editing').click();
+
+        const getTelemValuePromise = await subscribeToTelemetry(page, sineWaveObject.uuid);
+        await setStartOffset(page, { mins: '1' });
+        await setFixedTimeMode(page);
+
+        const subscribeTelemValue = await getTelemValuePromise;
+        const ladTableValuePromise = await page.waitForSelector(`text="${subscribeTelemValue}"`);
+        const ladTableValue = await ladTableValuePromise.textContent();
+
+        expect(ladTableValue).toBe(subscribeTelemValue);
+    });
+});
+
+/**
+ * Util for subscribing to a telemetry object by object identifier
+ * @param {import('@playwright/test').Page} page
+ * @param {string} objectIdentifier identifier for object
+ */
+async function subscribeToTelemetry(page, objectIdentifier) {
```

Comment:
This is awesome! Are there any current limitations? If so, we should make a note in the docs.

We can also specify the return value,

`@returns Promise<Promise<string>>`

Reply:
Sounds good!